pysces/vortex.py

```diff
@@ -79,6 +79,9 @@ def induced_velocity_single(self, x, xvort, gam):
         gam : float
             Strength of vortex
 
+        This method returns a vector of velocities at the points x induced
```

Comment:
Hi Clancy,

I've made all the changes previously suggested, including (simplistic)
tests of the new foil methods to increase code coverage.  I changed the
foil methods so that chord, rather than semichord, is consistently
specified.  I also adopted your good suggestion of making smaller, more
frequent commits.  Finally, regarding the docstrings, there are some
inconsistencies between PEP-0257 and the numpy docstring conventions.  To
select from these two, I took a cue from your code in body.py and went with
the numpy docstring conventions.

After committing all my changes and testing them, I pushed up to my fork
and verified that the build, coverage, and Travis tests succeeded.

P.S.: For the Joukowski and Karman-Trefftz foils, the parameter "a" only
determines the trailing edge.  Due to its nonuniform shape/camber, the
leading edge is determined (I believe) by a combination of the parameters.
This explains why I didn't call "a" the quarterchord, although for the
reasonable parameter sets I tried it is approximately so.  By playing
around on paper with the conformal map, I can chase the details down and
document it if you wish.

P.P.S: It goes without saying that adherence to these docstring conventions
needs to be implemented in the rest of the existing code as well.  That
will take some time.  As I read, understand, and modify code going forward,
I'll add/update docstrings according to the PEP-0257 and numpy docstring
conventions.

On Sun, Nov 22, 2015 at 10:31 AM, Clancy Rowley <notifications@github.com>
wrote:

> In pysces/vortex.py
> <https://github.com/cwrowley/pysces/pull/2#discussion_r45560186>:
>
> > @@ -79,6 +79,9 @@ def induced_velocity_single(self, x, xvort, gam):
> >          gam : float
> >              Strength of vortex
> >
> > +        This method returns a vector of velocities at the points x induced
>
> Use numpydoc
> <https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt>
> conventions for the returned values.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cwrowley/pysces/pull/2/files#r45560186>.
>